content/courses/advanced-ada/chapters/exceptions.rst

```diff
@@ -6,13 +6,63 @@ Exceptions
 Asserts
 -------
 
-.. admonition:: Relevant topics
+When we want to indicate a condition in the code that must always be valid, we
+can use the pragma :ada:`Assert`. As the name implies, when we use this pragma,
+we're *asserting* some truth about the source-code.
```

Comment:
What about the procedural form, via procedure Ada.Assertions.Assert?

Reply:
Good point, I've just added a source-code example with that procedure to the section.